packages/website/src/bin/page-generator/test.js

```diff
@@ -30,35 +32,42 @@ async function runGeneratePages({
         },
       ];
 
+  const resolvedPackageRoot = packageRoot || (await createTempDir());
+
+  const tempDefaultPagesPath = path.join(resolvedPackageRoot, 'default-pages');
+  await fs.copy(defaultPagesPath, tempDefaultPagesPath);
```

Comment:
Why do we need to copy the default-pages across, doesn't `generatePages` do that itself?

Reply:
This copies the default pages into the test directory which we generate pages in - if they are not there, generatePages fails because it can't copy them across.